go.mod

```diff
@@ -5,25 +5,25 @@ go 1.18
 require (
 	github.com/container-storage-interface/spec v1.6.0
 	github.com/evanphx/json-patch v4.12.0+incompatible
-	github.com/fsnotify/fsnotify v1.4.9
+	github.com/fsnotify/fsnotify v1.6.0
 	github.com/golang/mock v1.6.0
 	github.com/golang/protobuf v1.5.2
 	github.com/google/gofuzz v1.2.0
 	github.com/kubernetes-csi/csi-lib-utils v0.11.0
 	github.com/kubernetes-csi/csi-test/v4 v4.0.2
 	github.com/kubernetes-csi/external-snapshotter/client/v6 v6.1.0
-	github.com/prometheus/client_golang v1.12.1
-	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.32.1
-	github.com/spf13/cobra v1.4.0
-	google.golang.org/grpc v1.47.0
-	k8s.io/api v0.25.2
-	k8s.io/apimachinery v0.25.2
-	k8s.io/client-go v0.25.2
-	k8s.io/component-base v0.25.2
-	k8s.io/component-helpers v0.25.2
+	github.com/prometheus/client_golang v1.13.1
+	github.com/prometheus/client_model v0.3.0
+	github.com/prometheus/common v0.37.0
+	github.com/spf13/cobra v1.6.1
+	google.golang.org/grpc v1.50.1
+	k8s.io/api v0.25.3
+	k8s.io/apimachinery v0.25.3
+	k8s.io/client-go v1.5.2
```

Comment:
@humblec Can you fix this?  This is an old version.

Reply:
I submitted a PR: https://github.com/kubernetes-csi/external-snapshotter/pull/785